MAUI/DataForm/DataForm-settings.md

```diff
@@ -0,0 +1,854 @@
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null)
+    {
+        if (e.DataFormItem.FieldName == "PhoneNumer")
+        {
+            e.DataFormItem.LeadingLabelIcon = new FontImageSource() 
+            { 
+                Glyph = "K", 
+                Color = Colors.Black, 
+                FontFamily = "DataFormIcons", 
+                Size=18 
+            };
+        }
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+
+## Change the Text style 
+
+You can change the text styles of the label, editor, valid message, and error message by using the `GenereateDataFormItem` event when `AutoGenerateItems` is true.
+
+<table>
+<tr>
+<th>Text</th>
+<th>Property</th>
+</tr>
+<tr>
+<td>Label Text</td>
+<td>LabelTextStyle</td>
+</tr>
+<tr>
+<td>Editor Text</td>
+<td>EditorTextStyle</td>
+</tr>
+<tr>
+<td>Error Message Text</td>
+<td>ErrorLabelTextStyle</td>
+</tr>
+<tr>
+<td>Valid Message Text</td>
+<td>ValidMessageLabelTextStyle</td>
+</tr>
+</table>
+
+By using `DataFormTextStyle` class properties, you can change the text style .
+
+DataFormTextStyle class properties:-
+    1.FontSize
+    2.FontFamily
+    3.FontAttributes
+    4.TextColor
+
+**Change Label Text Style**
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem.FieldName == "Name" && e.DataFormItem is DataFormTextItem textItem)
+    {
+        e.DataFormItem.LabelTextStyle = new DataFormTextStyle() 
+        {
+            FontSize = 19, 
+            TextColor = Colors.Pink, 
+            FontAttributes = FontAttributes.Italic
+        };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+**Change Editor Text Style**
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem.FieldName == "Name" && e.DataFormItem is DataFormTextItem textItem)
+    {
+        e.DataFormItem.EditorTextStyle = new DataFormTextStyle() 
+        {
+            FontSize = 19, 
+            TextColor = Colors.Blue, 
+            FontAttributes = FontAttributes.Italic
+        };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+Change valid Message Text style,
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.ValidMessageLabelTextStyle = new DataFormTextStyle() 
+        {
+            FontSize = 19, 
+            TextColor = Colors.Blue, 
+            FontAttributes = FontAttributes.Bold
+        };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+Change error message text style,
```

Comment:
Change error message text style,

Remove this topic as explained in validation topic

Reply:
removed